policy/templates/releng/ci/goreleaser/goreleaser.yml

```diff
@@ -28,7 +28,7 @@ checksum:
   disable: true
 
 release:
-  disable: true
+  disable: tr221
```

Comment:
?

Reply:
Hi Alok, yes is actually in development I will turn this into draft until is ready for review